roles/forklift_versions/defaults/main.yml

```diff
@@ -1,2 +1,2 @@
-forklift_versions_file: "{{ playbook_dir | dirname }}/vagrant/config/versions.yaml"
+forklift_versions_file: "{{ lookup('env', 'PWD') }}/vagrant/config/versions.yaml"
```

Comment:
Why is this needed? In my experience this failed because the directory changed. Should we use `forklift_directory` here?

Reply:
This existing code puts on a restriction that playbooks can only be a single directory deep. I seem to recall PWD causing some issue in the past but not what issue. So far my testing has been all successful including my testing inside `ci.centos.org`.